CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - The environment variable used for resource detection has been changed from `OTEL_RESOURCE_LABELS` to `OTEL_RESOURCE_ATTRIBUTES` (#1042)
 - Replace `WithSyncer` with `WithBatcher` in examples. (#1044)
 - Replace the `google.golang.org/grpc/codes` dependency in the API with an equivalent `go.opentelemetry.io/otel/codes` package. (#1046)
+- Integration tests for OTel Collector Attributes. (#1062)
```

Comment:
```suggestion
- Integration tests for more OTel Collector Attribute types. (#1062)
```

I'd also suggest moving this to an `Added` section between the `[Unreleased]` section marker and the `Changed` section marker as these are wholly new integration tests.

Reply:
Thanks. fixed at lastest commit.